### PR TITLE
Wip: Add Material::from(isize)

### PR DESCRIPTION
**Pull request by kepet19**
Hello this is a work in progress, 
I don't know if you want a huge match statement like this or you would like to use something else.

When I run it from minecraft server 1.16.4 
it tries too set item 902. but is is not in the list.
image
image

do somebody know where to look for minecraf block id's? 

I am going to remove log::info later. it is just for debugging purpose.

### CODE REVIEW

**Comment by PureTryOut** [comment]
> I am going to remove log::info later. it is just for debugging purpose

Consider using `log::debug`, it´s made for such things :wink:

File: src/server/mod.rs [hunk 1]
```diff
@@ -689,8 +689,8 @@ impl Server {
                                 set_slot.item.as_ref().map_or(0, |s| s.id)
                             );
                             let item = set_slot.item.map(|stack| Item {
+                                material: Material::from(stack.id), // TODO: map stack.id to material!
```
Comment (PureTryOut) [inline src/server/mod.rs:692]: Since you´re resolving the todo, remove the todo :wink:

**Comment by terrarier2111** [comment]
I am already implementing this with cross version compatability, i'll be done by today evening (in round about 8hrs)

**Comment by kepet19** [comment]
> > I am going to remove log::info later. it is just for debugging purpose
> 
> Consider using `log::debug`, it´s made for such things wink
> 
> > do somebody know where to look for minecraf block id's?
> 
> Almost everything you need is documented on the Minecraft wiki.
> 
> For block ID´s up-to 1.12: https://minecraft.fandom.com/wiki/Java_Edition_data_values/Pre-flattening
> Since 1.13: https://minecraft.fandom.com/wiki/Java_Edition_data_values

You would be fine to leave it with log::debug ? 

I am not sure I tried yesterday with log:debug, I could not get it out to the terminal. I tried it with Environment variable `RUST_LOG=debug`, could not get that to work. not sure how to debug.

**Comment by kepet19** [comment]
> I am already implementing this with cross version compatability, i'll be done by today evening (in round about 8hrs)

okay that sounds nice. I am just gonna close this issue.

**Comment by PureTryOut** [comment]
> I tried it with Environment variable RUST_LOG=debug, could not get that to work. not sure how to debug.

Hmm that should be enough, not sure then.

**Pull request closed without merging**